lib/invoice_repo.rb

```diff
@@ -20,4 +21,40 @@ def find_all_by_merchant_id(id)
   def find_all_by_status(status)
     @repository.select { |invoice| invoice.status == status }
   end
+
+  def invoice_status(type)
```

Comment:
I know this is the name the iterations has us use, and so the SA should still call it, however I find this name misleading, I am surprised every time I remember it is returning a percent.  Maybe we can change it at the level of the repo. Not important though.